schema.js

```diff
@@ -81,6 +81,15 @@ const schema = buildSchema(`
     hasDBLounge: Boolean!
     hasLostAndFound: Boolean!
     hasCarRental: Boolean!
+    tracks: [Track!]!
+  }
+
+  type Track {
+	  platform: String!
+	  number: String!
+	  name: String!
+	  length: Int
```

Comment:
See Platforms/Track.js: Are values for length and height Int or Float? Check incoming data!